src/components/shared/Util.js

```diff
@@ -8,6 +8,18 @@ const groupBy = (list, props) => {
     }, []);
 }
 
+const colors = [  // darker colors
```

Comment:
Just noticed that there are 10 races and 9 classes (because nobody created yet a DK on the server).

You might want to define the color with index 9 too, otherwise you get that default light blue.

Reply:
Also, I prefer more this colors, instead of the ones that were commented out.

Reply:
we need to make a generic application also for others AC servers, so we could put 10 colors ;-)

Reply:
so you like the darker?